src/chrome/content/rules/additifs-alimentaires.net.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="additifs-alimentaires.net">
+	<target host="additifs-alimentaires.net" />
+	<target host="www.additifs-alimentaires.net" />
+
```

Comment:
`<securecookie host=".+" name=".+" />`

Reply:
Cf. contributor guide : `For securecookie tags, if you know that all cookies on the included targets can be secured (which in particular means that the cookies are not used by any of its non-securable subdomains)`. How do you know it's the case ? Or is it now enforced for every new ruleset ?

Reply:
@Harest Generally, if `<securecookie>` doesn't cause any issues, it should be added.

Reply:
Okay, i'll keep that in mind and add it then. Thanks for the info.